src/language-html/printer-html.js

```diff
@@ -1033,6 +1033,11 @@ function printEmbeddedAttributeValue(node, originalTextToDoc, options) {
      *     bindon-target="angularExpression"
      */
     const ngExpressionBindingPatterns = ["^\\[.+\\]$", "^bind(on)?-"];
+    /**
+     *     i18n="longDescription"
+     *     i18n-attr="longDescription"
```

Comment:
I’m not familiar with Angular. Do these kinds of attributes only contain plain text, or could they also contain expressions or code of some sort? If so, we should add a test for that.

Reply:
That's a great point - they don't contain code, but they can contain ID syntax as well as "meaning" syntax, denoted by `@@` and `|` separators. Added some test cases.